server.js

```diff
@@ -2,65 +2,40 @@
 const express = require('express');
 const app = express();
 
-// run `npm install` to install dependencies in package.json
+const MarsMissions = require('./models/missions');
 
 // * Your mission is to complete the app
 // * The app will need routes for index and show
+
+//Index route - missions
+app.get('/missions', (req, res) => {
+  res.send(MarsMissions);
+});
+
+//show route - missions
+app.get('/missions/:id', (req, res) => {
+  res.render('show.ejs', {
+    marsMissions: MarsMissions[req.param.id]
```

Comment:
this looks good but the template you're rendering doesn't exist...... make it!

Reply:
and make a `views` directory to put it in :)

Reply:
!!This just in! on closer inspection, i see that you have made views, but the folder they're in is called "missions."  change that folder name to `views`.